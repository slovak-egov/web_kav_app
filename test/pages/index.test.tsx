import React from 'react';
import { DrupalNode } from 'next-drupal';
import { render } from '../testUtils';
import IndexPage, { MenuData, PathObject } from '../../pages/index';

describe('IndexPage', () => {
  it('matches snapshot', () => {
    const mockNode = {
      type: 'node--article',
      id: '222d13ee-a245-49c5-8db1-a862a1ec6a13',
      drupal_internal__nid: 1,
      drupal_internal__vid: 10,
      langcode: 'sk',
      revision_timestamp: '2022-06-30T13:11:31+00:00',
      revision_log: null,
      status: true,
      title: 'Hello world Hello world',
      created: '2022-06-30T13:08:26+00:00',
      changed: '2022-06-30T13:11:23+00:00',
      promote: false,
      sticky: false,
      default_langcode: true,
      revision_translation_affected: true,
      moderation_state: 'published',
      path: {
        alias: null,
        pid: null,
        langcode: 'sk'
      },
      content_translation_source: 'und',
      content_translation_outdated: false,
      comment: {
        status: 2,
        cid: 0,
        last_comment_timestamp: 1656594524,
        last_comment_name: null,
        last_comment_uid: 1,
        comment_count: 0
      },
      field_body: null,
      field_perex: {
        value: 'Lorem ipsum',
        format: 'plain_text',
        processed: '<p>Lorem ipsum</p>\n'
      },
      links: {
        self: {
          href: 'http://cms-slovensko-sk.lndo.site/jsonapi/node/article/222d13ee-a245-49c5-8db1-a862a1ec6a13?resourceVersion=id%3A10'
        }
      },
      node_type: null,
      revision_uid: null,
      uid: null,
      field_image: {
        links: {
          self: { href: 'https://rgw.slovenskoit.sk/svm.dev-cms/public/images/Logo_0.png' }
        }
      },
      field_tags: [],
      relationshipNames: ['node_type', 'revision_uid', 'uid', 'field_image']
    } as unknown as DrupalNode;
    const mockLayoutData = {
      type: 'config--general',
      id: 'de71f69c-0255-49ed-9ff2-00e83629fbaa',
      egovHeaderTriggerLink1: 'Oficiálna stránka',
      egovHeaderTriggerLink2: 'verejnej správy SR',
      egovHeaderTitle1: 'Doména gov.sk je oficálna',
      egovHeaderText1: {
        value:
          '<p>Toto je oficiálna webová stránka orgánu verejnej moci Slovenskej republiky. Oficiálne stránky využívajú najmä doménu gov.sk. <a href="https://www.slovensko.sk/sk/agendy/agenda/_organy-verejnej-moci">Odkazy na jednotlivé webové sídla orgánov verejnej moci nájdete na tomto odkaze.</a></p>\r\n',
        format: 'basic_html'
      },
      egovHeaderTitle2: 'Táto stránka je zabezpečená',
      egovHeaderText2: {
        value:
          '<p>Buďte pozorní a vždy sa uistite, že zdieľate informácie iba cez zabezpečenú webovú stránku verejnej správy SR. Zabezpečená stránka vždy začína https:// pred názvom domény webového sídla.</p>\r\n',
        format: 'basic_html'
      },
      headerLogo: 'https://rgw.slovenskoit.sk/svm.dev-cms/public/images/Logo_0.png',
      footerClaim: {
        value:
          '<p><span>Prevádzkovateľom služby je Ministerstvo investícií, regionálneho rozvoja a informatizácie SR.</span></p>\r\n\r\n<p><span>Vytvorené v súlade s Jednotným dizajn manuálom elektronických služieb.</span></p>\r\n',
        format: 'basic_html'
      },
      footerLogo: 'https://rgw.slovenskoit.sk/svm.dev-cms/public/images/Logo%281%29.png'
    } as unknown as DrupalNode;
    const mockMenuData = {
      header: [
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:c62ec868-1044-46e9-9ae9-66ebb1f7ee30',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'main',
          meta: {
            entity_id: '1'
          },
          options: [],
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: 'entity.node.canonical',
            parameters: {
              node: '1'
            }
          },
          title: 'Test',
          url: '/clanok-1',
          weight: '0'
        },
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:6afda2c4-d819-4bf8-b994-48bc7ae077cb',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'main',
          meta: {
            entity_id: '2'
          },
          options: {
            query: []
          },
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: 'entity.node.canonical',
            parameters: {
              node: '3'
            }
          },
          title: 'ZU',
          url: '/test-test',
          weight: '0'
        },
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:f29c78bb-2a22-47ed-9c51-426cf0853e81',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'main',
          meta: {
            entity_id: '3'
          },
          options: [],
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: '<nolink>',
            parameters: []
          },
          title: 'Drop Down',
          url: '',
          weight: '1',
          items: [
            {
              type: 'menu_link_content--menu_link_content',
              id: 'menu_link_content:b27fec8a-2323-4faf-8ed8-72b3ab216958',
              description: null,
              enabled: true,
              expanded: false,
              menu_name: 'main',
              meta: {
                entity_id: '4'
              },
              options: {
                query: []
              },
              parent: 'menu_link_content:f29c78bb-2a22-47ed-9c51-426cf0853e81',
              provider: 'menu_link_content',
              route: {
                name: 'entity.node.canonical',
                parameters: {
                  node: '1'
                }
              },
              title: 'option 1',
              url: '/clanok-1',
              weight: '0'
            },
            {
              type: 'menu_link_content--menu_link_content',
              id: 'menu_link_content:e6077154-31ef-43d0-a9e6-08ec58d1d1d4',
              description: null,
              enabled: true,
              expanded: false,
              menu_name: 'main',
              meta: {
                entity_id: '5'
              },
              options: {
                query: []
              },
              parent: 'menu_link_content:f29c78bb-2a22-47ed-9c51-426cf0853e81',
              provider: 'menu_link_content',
              route: {
                name: 'entity.node.canonical',
                parameters: {
                  node: '1'
                }
              },
              title: 'option 2',
              url: '/clanok-1',
              weight: '0'
            }
          ]
        }
      ],
      footer: [
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:743679ca-f253-451d-aa8a-1b785262d6ec',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'footer',
          meta: {
            entity_id: '6'
          },
          options: [],
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: '<nolink>',
            parameters: []
          },
          title: 'Informácie',
          url: '',
          weight: '0',
          items: [
            {
              type: 'menu_link_content--menu_link_content',
              id: 'menu_link_content:df77d784-5770-4f9a-908e-327cc16d15d5',
              description: null,
              enabled: true,
              expanded: false,
              menu_name: 'footer',
              meta: {
                entity_id: '9'
              },
              options: [],
              parent: 'menu_link_content:743679ca-f253-451d-aa8a-1b785262d6ec',
              provider: 'menu_link_content',
              route: {
                name: '',
                parameters: []
              },
              title: 'Informácie a návody',
              url: '/test',
              weight: '0'
            },
            {
              type: 'menu_link_content--menu_link_content',
              id: 'menu_link_content:8f81570e-002d-4321-863d-9344c094cc0c',
              description: null,
              enabled: true,
              expanded: false,
              menu_name: 'footer',
              meta: {
                entity_id: '10'
              },
              options: [],
              parent: 'menu_link_content:743679ca-f253-451d-aa8a-1b785262d6ec',
              provider: 'menu_link_content',
              route: {
                name: '',
                parameters: []
              },
              title: 'Pre občanov',
              url: '/test',
              weight: '0'
            }
          ]
        },
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:b9c44e8b-aaaf-427a-b9ef-19684c1a6582',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'footer',
          meta: {
            entity_id: '14'
          },
          options: [],
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: '<nolink>',
            parameters: []
          },
          title: '&nbsp;',
          url: '',
          weight: '1',
          items: [
            {
              type: 'menu_link_content--menu_link_content',
              id: 'menu_link_content:972c203b-1603-4af9-b3fa-6abbfa06ea55',
              description: null,
              enabled: true,
              expanded: false,
              menu_name: 'footer',
              meta: {
                entity_id: '16'
              },
              options: [],
              parent: 'menu_link_content:b9c44e8b-aaaf-427a-b9ef-19684c1a6582',
              provider: 'menu_link_content',
              route: {
                name: '',
                parameters: []
              },
              title: 'Informácie a návody',
              url: '/test',
              weight: '0'
            },
            {
              type: 'menu_link_content--menu_link_content',
              id: 'menu_link_content:6172700d-fd34-4a06-b9fc-59499b5c9c6a',
              description: null,
              enabled: true,
              expanded: false,
              menu_name: 'footer',
              meta: {
                entity_id: '15'
              },
              options: [],
              parent: 'menu_link_content:b9c44e8b-aaaf-427a-b9ef-19684c1a6582',
              provider: 'menu_link_content',
              route: {
                name: '',
                parameters: []
              },
              title: 'Často kladené otázky',
              url: '/test',
              weight: '0'
            }
          ]
        },
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:34bf58dc-6229-44e3-a6c5-a3b95cd978d4',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'footer',
          meta: {
            entity_id: '8'
          },
          options: [],
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: '<nolink>',
            parameters: []
          },
          title: 'Podpora',
          url: '',
          weight: '2',
          items: [
            {
              type: 'menu_link_content--menu_link_content',
              id: 'menu_link_content:4941b4d0-b605-4871-8a4f-85cbb30f8fe0',
              description: null,
              enabled: true,
              expanded: false,
              menu_name: 'footer',
              meta: {
                entity_id: '11'
              },
              options: [],
              parent: 'menu_link_content:34bf58dc-6229-44e3-a6c5-a3b95cd978d4',
              provider: 'menu_link_content',
              route: {
                name: '',
                parameters: []
              },
              title: 'Informácie a návody',
              url: '/test',
              weight: '0'
            },
            {
              type: 'menu_link_content--menu_link_content',
              id: 'menu_link_content:e8e647ec-0b8b-4c9d-a17b-cf118b5472db',
              description: null,
              enabled: true,
              expanded: false,
              menu_name: 'footer',
              meta: {
                entity_id: '12'
              },
              options: [],
              parent: 'menu_link_content:34bf58dc-6229-44e3-a6c5-a3b95cd978d4',
              provider: 'menu_link_content',
              route: {
                name: '',
                parameters: []
              },
              title: 'Pre občanov',
              url: '/test',
              weight: '0'
            }
          ]
        },
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:756d9bac-adf9-4fa6-84c3-9416f74a1dc0',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'footer',
          meta: {
            entity_id: '7'
          },
          options: [],
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: '<nolink>',
            parameters: []
          },
          title: 'Rýchle odkazy',
          url: '',
          weight: '3',
          items: [
            {
              type: 'menu_link_content--menu_link_content',
              id: 'menu_link_content:ca37a25a-baf4-4b93-8f11-980368c236e2',
              description: null,
              enabled: true,
              expanded: false,
              menu_name: 'footer',
              meta: {
                entity_id: '13'
              },
              options: [],
              parent: 'menu_link_content:756d9bac-adf9-4fa6-84c3-9416f74a1dc0',
              provider: 'menu_link_content',
              route: {
                name: '',
                parameters: []
              },
              title: 'Často kladené otázky',
              url: '/test',
              weight: '0'
            }
          ]
        }
      ],
      footerAside: [
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:a05c0df1-d9d2-4229-ab64-16f39328faa1',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'footer-aside',
          meta: {
            entity_id: '17'
          },
          options: [],
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: '',
            parameters: []
          },
          title: 'Mapa stránok',
          url: '/test',
          weight: '0'
        },
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:c589e792-045b-47a8-8a73-5f9b788fdf54',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'footer-aside',
          meta: {
            entity_id: '19'
          },
          options: [],
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: '',
            parameters: []
          },
          title: 'Ochrana osobných údajov',
          url: '/test',
          weight: '0'
        },
        {
          type: 'menu_link_content--menu_link_content',
          id: 'menu_link_content:88f5adef-2549-459f-921b-46d91b3808cf',
          description: null,
          enabled: true,
          expanded: false,
          menu_name: 'footer-aside',
          meta: {
            entity_id: '18'
          },
          options: [],
          parent: '',
          provider: 'menu_link_content',
          route: {
            name: '',
            parameters: []
          },
          title: 'RSS',
          url: '/test',
          weight: '0'
        }
      ]
    } as unknown as MenuData;
    const mockPath = {
      entity: {
        aliases: {
          sk: '/',
          en: '/'
        }
      }
    } as PathObject;
    const { asFragment } = render(
      <IndexPage
        menuData={mockMenuData}
        node={mockNode}
        layoutData={mockLayoutData}
        path={mockPath}
      />,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
