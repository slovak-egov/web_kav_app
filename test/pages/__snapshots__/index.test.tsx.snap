// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="page-layout"
  >
    <div
      data-focus-guard="true"
      style="width: 1px; height: 0px; padding: 0px; overflow: hidden; position: fixed; top: 1px; left: 1px;"
      tabindex="-1"
    />
    <div
      data-focus-lock-disabled="disabled"
    >
      <header
        class="header-container__wrapper sticky"
      >
        <div
          class="secondary-navigation"
        >
          <div
            class="secondary-navigation__header"
          >
            <div
              class="secondary-navigation__heading"
            >
              Oficiálna stránka 
              <button
                class="secondary-navigation__heading-button"
              >
                verejnej správy SR 
                <svg
                  class="secondary-navigation__heading-button-icon"
                  fill="none"
                  height="1em"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="m8.71 11.71 2.59 2.59c.39.39 1.02.39 1.41 0l2.59-2.59c.63-.63.18-1.71-.71-1.71H9.41c-.89 0-1.33 1.08-.7 1.71Z"
                    fill="currentColor"
                  />
                </svg>
              </button>
            </div>
            <div
              class="dropdown__wrapper secondary-navigation__dropdown"
            >
              <button
                class="dropdown"
              >
                <span>
                  language.undefined
                </span>
                <svg
                  class="dropdown__icon"
                  fill="none"
                  height="1em"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="m8.71 11.71 2.59 2.59c.39.39 1.02.39 1.41 0l2.59-2.59c.63-.63.18-1.71-.71-1.71H9.41c-.89 0-1.33 1.08-.7 1.71Z"
                    fill="currentColor"
                  />
                </svg>
              </button>
              <ul
                class="dropdown__options hidden"
                data-testid="dropdown-options"
              />
            </div>
          </div>
          <div
            class="secondary-navigation__body hidden"
            data-testid="secnav-children"
          >
            <div
              class="grid grid-cols-1 gap-4 tb2:grid-cols-2 tb2:gap-8"
              id="secondary-navigation-body"
            >
              <div>
                <h3
                  class="idsk-text-body-1"
                >
                  Doména gov.sk je oficálna
                </h3>
                <div
                  class="py-2.5"
                >
                  <p>
                    Toto je oficiálna webová stránka orgánu verejnej moci Slovenskej republiky. Oficiálne stránky využívajú najmä doménu gov.sk. 
                    <a
                      href="https://www.slovensko.sk/sk/agendy/agenda/_organy-verejnej-moci"
                    >
                      Odkazy na jednotlivé webové sídla orgánov verejnej moci nájdete na tomto odkaze.
                    </a>
                  </p>
                  

                </div>
              </div>
              <div>
                <h3
                  class="idsk-text-body-1"
                >
                  Táto stránka je zabezpečená
                </h3>
                <div
                  class="py-2.5"
                >
                  <p>
                    Buďte pozorní a vždy sa uistite, že zdieľate informácie iba cez zabezpečenú webovú stránku verejnej správy SR. Zabezpečená stránka vždy začína https:// pred názvom domény webového sídla.
                  </p>
                  

                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="header-container"
        >
          <a
            class="inline-block h-[1.875rem] tb1:h-10 w-60 relative"
            href="/"
          >
            <span
              style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
            >
              <img
                data-nimg="fill"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain; object-position: left;"
              />
              <noscript />
            </span>
          </a>
          <a
            class="icon-link dm1:hidden hidden"
            href="/"
          >
            <test-file-stub
              classname="h-6"
            />
          </a>
          <div
            class="hidden dm1:flex flex-auto items-center h-full text-blue-600"
          >
            <div
              class="flex-auto"
            />
            <div
              class="mx-5 hidden dm1:flex"
            >
              <div
                class="searchbar__wrapper"
              >
                <input
                  class="searchbar searchbar--medium w-72 max-w-96 pr-10"
                  placeholder="header_search.placeholder"
                />
                <button
                  class="searchbar__button searchbar__button--medium"
                >
                  <svg
                    class="searchbar__icon--medium"
                    fill="none"
                    height="1em"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M15.5 14h-.79l-.28-.27a6.5 6.5 0 0 0 1.48-5.34c-.47-2.78-2.79-5-5.59-5.34a6.505 6.505 0 0 0-7.27 7.27c.34 2.8 2.56 5.12 5.34 5.59a6.5 6.5 0 0 0 5.34-1.48l.27.28v.79l4.25 4.25c.41.41 1.08.41 1.49 0 .41-.41.41-1.08 0-1.49L15.5 14Zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14Z"
                      fill="currentColor"
                    />
                  </svg>
                </button>
              </div>
            </div>
            <button
              class="primary-button primary-button--basic primary-button--medium"
            >
              Login
            </button>
          </div>
          <div
            class="flex-auto dm1:hidden"
          />
          <button
            class="menu-button dm1:hidden"
          >
            <span>
              Menu
            </span>
             
            <svg
              class="menu-button__icon"
              fill="none"
              height="1em"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M4 18h16c.55 0 1-.45 1-1s-.45-1-1-1H4c-.55 0-1 .45-1 1s.45 1 1 1Zm0-5h16c.55 0 1-.45 1-1s-.45-1-1-1H4c-.55 0-1 .45-1 1s.45 1 1 1ZM3 7c0 .55.45 1 1 1h16c.55 0 1-.45 1-1s-.45-1-1-1H4c-.55 0-1 .45-1 1Z"
                fill="currentColor"
              />
            </svg>
          </button>
        </div>
        <div
          class="header-container__menu"
        >
          <nav
            class="navigation navigation--full"
          >
            <span
              class="navigation__link"
            >
              <a
                href="/"
              >
                Home
              </a>
            </span>
            <span
              class="navigation__link"
            >
              <a
                href="/clanok-1"
              >
                Test
              </a>
            </span>
            <span
              class="navigation__link"
            >
              <a
                href="/test-test"
              >
                ZU
              </a>
            </span>
            <div
              class="dropdown__wrapper navigation__dropdown"
            >
              <button
                class="dropdown dm1:px-3"
              >
                <span>
                  Drop Down
                </span>
                <svg
                  class="dropdown__icon"
                  fill="none"
                  height="1em"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M15.88 9.29 12 13.17 8.12 9.29a.996.996 0 1 0-1.41 1.41l4.59 4.59c.39.39 1.02.39 1.41 0l4.59-4.59a.996.996 0 0 0 0-1.41c-.39-.38-1.03-.39-1.42 0Z"
                    fill="currentColor"
                  />
                </svg>
              </button>
              <ul
                class="dropdown__options hidden navigation__dropdown-options"
                data-testid="dropdown-options"
              >
                <li
                  class="dropdown__option"
                >
                  <a
                    href="/clanok-1"
                  >
                    option 1
                  </a>
                </li>
                <li
                  class="dropdown__option"
                >
                  <a
                    href="/clanok-1"
                  >
                    option 2
                  </a>
                </li>
              </ul>
            </div>
          </nav>
        </div>
        <div
          class="header-container__menu--mobile"
        >
          <div
            class="menu--mobile hidden"
          >
            <nav
              class="navigation"
            >
              <span
                class="navigation__label"
              >
                Menu
              </span>
              <span
                class="navigation__link"
              >
                <a
                  href="/"
                >
                  Home
                </a>
              </span>
              <span
                class="navigation__link"
              >
                <a
                  href="/clanok-1"
                >
                  Test
                </a>
              </span>
              <span
                class="navigation__link"
              >
                <a
                  href="/test-test"
                >
                  ZU
                </a>
              </span>
              <div
                class="dropdown__wrapper navigation__dropdown"
              >
                <button
                  class="dropdown dm1:px-3"
                >
                  <span>
                    Drop Down
                  </span>
                  <svg
                    class="dropdown__icon"
                    fill="none"
                    height="1em"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M15.88 9.29 12 13.17 8.12 9.29a.996.996 0 1 0-1.41 1.41l4.59 4.59c.39.39 1.02.39 1.41 0l4.59-4.59a.996.996 0 0 0 0-1.41c-.39-.38-1.03-.39-1.42 0Z"
                      fill="currentColor"
                    />
                  </svg>
                </button>
                <ul
                  class="dropdown__options hidden navigation__dropdown-options"
                  data-testid="dropdown-options"
                >
                  <li
                    class="dropdown__option"
                  >
                    <a
                      href="/clanok-1"
                    >
                      option 1
                    </a>
                  </li>
                  <li
                    class="dropdown__option"
                  >
                    <a
                      href="/clanok-1"
                    >
                      option 2
                    </a>
                  </li>
                </ul>
              </div>
            </nav>
            <button
              class="primary-button primary-button--basic primary-button--medium w-full mt-4"
            >
              Login
            </button>
          </div>
        </div>
      </header>
    </div>
    <div
      data-focus-guard="true"
      style="width: 1px; height: 0px; padding: 0px; overflow: hidden; position: fixed; top: 1px; left: 1px;"
      tabindex="-1"
    />
    <main
      class="page-layout__main bg-white gap-6 ml-"
    >
      <div
        class="page-layout__title"
      />
      <div
        class="page-layout__body"
      >
        <div>
          <h1
            class="mb-10 text-6xl font-black"
          >
            Aktuality
          </h1>
          <p
            class="mb-10"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque molestie, justo lacinia hendrerit posuere, ipsum ipsum lobortis dui, aliquam pulvinar ex felis at nunc. Aenean eu felis sem. Vivamus eu mauris non orci ullamcorper ultricies quis sed leo. Vivamus non velit sed diam aliquam ornare.
          </p>
          <div>
            <div
              class="mb-10 flex gap-2.5 flex-wrap"
            />
            <div
              class="mb-10"
            >
              <article
                class="mb-4"
              >
                <div
                  class="idsk-anchor-card idsk-anchor-card--vertical idsk-anchor-card--grid"
                  data-testid="anchor-card"
                >
                  <div
                    class="article-card__image-wrapper pt-0"
                  >
                    <span
                      style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
                    >
                      <span
                        style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 62.5% 0px 0px 0px;"
                      />
                      <img
                        alt=""
                        data-nimg="responsive"
                        decoding="async"
                        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                        style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
                      />
                      <noscript />
                    </span>
                  </div>
                  <div
                    class="flex-1"
                  >
                    <p
                      class="article-card__date-tags--top"
                    >
                      <time
                        datetime="30.06.2022"
                      >
                        30.06.2022
                      </time>
                       
                    </p>
                    <a
                      class="idsk-anchor-card__heading"
                    >
                      Hello world Hello world
                    </a>
                    <div
                      class="article-card__text"
                    >
                      <p>
                        Lorem ipsum
                      </p>
                      

                    </div>
                  </div>
                </div>
              </article>
            </div>
            <ul
              class="flex items-center text-primary gap-5"
            >
              <li
                class="previous disabled"
              >
                <a
                  aria-disabled="true"
                  aria-label="Previous page"
                  class="idsk-link-m "
                  rel="prev"
                  role="button"
                  tabindex="-1"
                >
                  Preview
                </a>
              </li>
              <li
                class="selected"
              >
                <a
                  aria-current="page"
                  aria-label="Page 1 is your current page"
                  class="idsk-link-m text-blue-600 font-bold no-underline"
                  rel="canonical"
                  role="button"
                  tabindex="-1"
                >
                  1
                </a>
              </li>
              <li
                class="next disabled"
              >
                <a
                  aria-disabled="true"
                  aria-label="Next page"
                  class="idsk-link-m "
                  rel="next"
                  role="button"
                  tabindex="-1"
                >
                  Next
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="footer-container"
    >
      <div
        class="footer-container__content"
      >
        <div
          class="grid grid-cols-1 tb1:grid-cols-2 tb2:grid-cols-4 gap-5"
        >
          <div
            class="footer-container__section"
          >
            <p
              class="footer-container__section-heading"
            >
              Informácie
            </p>
            <a
              href="/test"
            >
              Informácie a návody
            </a>
            <a
              href="/test"
            >
              Pre občanov
            </a>
          </div>
          <div
            class="footer-container__section"
          >
            <p
              class="footer-container__section-heading hidden tb1:block"
            >
               
            </p>
            <a
              href="/test"
            >
              Informácie a návody
            </a>
            <a
              href="/test"
            >
              Často kladené otázky
            </a>
          </div>
          <div
            class="footer-container__section"
          >
            <p
              class="footer-container__section-heading"
            >
              Podpora
            </p>
            <a
              href="/test"
            >
              Informácie a návody
            </a>
            <a
              href="/test"
            >
              Pre občanov
            </a>
          </div>
          <div
            class="footer-container__section tb2:items-end"
          >
            <p
              class="footer-container__section-heading"
            >
              Rýchle odkazy
            </p>
            <a
              href="/test"
            >
              Často kladené otázky
            </a>
          </div>
        </div>
        <aside
          class="footer-container__aside"
        >
          <div
            class="footer-container__links-list"
          >
            <a
              href="/test"
            >
              Mapa stránok
            </a>
            <a
              href="/test"
            >
              Ochrana osobných údajov
            </a>
            <a
              href="/test"
            >
              RSS
            </a>
          </div>
          <div
            class="flex justify-between flex-wrap gap-8"
          >
            <div>
              <p>
                <span>
                  Prevádzkovateľom služby je Ministerstvo investícií, regionálneho rozvoja a informatizácie SR.
                </span>
              </p>
              


              <p>
                <span>
                  Vytvorené v súlade s Jednotným dizajn manuálom elektronických služieb.
                </span>
              </p>
              

            </div>
            <div
              class="relative w-40 flex items-center"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27162%27%20height=%2736%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
          </div>
        </aside>
      </div>
    </footer>
  </div>
</DocumentFragment>
`;
